bread/bread.py

```diff
@@ -362,9 +362,21 @@ def construct_search(field_name):
             if not use_distinct:
                 opts = self.bread.model._meta
                 for search_spec in orm_lookups:
-                    if lookup_needs_distinct(opts, search_spec):
-                        use_distinct = True
-                        break
+                    # The function lookup_needs_distinct() was renamed
+                    # to lookup_spawns_duplicates() in Django 4.0
+                    # https://docs.djangoproject.com/en/4.2/releases/4.0/#:~:text=The%20undocumented%20django.contrib.admin.utils.lookup_needs_distinct()%20function%20is%20renamed%20to%20lookup_spawns_duplicates().
+                    try:
+                        from django.contrib.admin.utils import lookup_needs_distinct
```

Comment:
Nitpick: Try the new function first, then fall back to the older version if needed.